### PR TITLE
Coreutils : created basic cp command

### PR DESCRIPTION
**Pull request by Kaushal1011**
created basic cp command in coreutils. It uses read and write stream to copy contents of one file to another (that is newly created).

### CODE REVIEW

File: applications/coreutils/cp.cpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+// - Create new file
+// - append to already existing file
+// - only copy if file already exists
+// - only logs if used with option -v
+
+int main(int argc, char **argv)
+{
+    if (argc == 3)
+    {
+        // open input stream
```
Comment (sleepy-monax) [inline applications/coreutils/cp.cpp:16]: Please avoid comment that paraphrases the code. Comments should answer the why and the code the how :)

Comment (Kaushal1011) [reply applications/coreutils/cp.cpp]: removed the comments :)

**Comment by sleepy-monax** [comment]
Nice work :)

**Comment by Kaushal1011** [comment]
i have written close stream at the end of file which invokes warning for freeing resource twice (._.'), I just fixed it as well, i think the merge was done before my push

**Pull request merged**